upi/openstack/update_ign.py

```diff
@@ -155,7 +156,7 @@ def update(hostname,ignition):
 DEFROUTE=no
 PROXY_METHOD=none
 BROWSER_ONLY=no
-MTU=""" + neutron_network_mtu + """
+MTU=""" + opflex_network_mtu + """
```

Comment:
Perhaps I am missing something, why are we changing this?

Reply:
Sumit this is according to cams comments where we are modifying the opflex interface mtu according to what is mentioned in inventory.yaml

Reply:
Cam's comment?? Where/what? This is the node network, why are we changing this to opflex mtu?